StandardLibrary/runtimes/java/build.gradle.kts

```diff
@@ -68,12 +67,7 @@ publishing {
         artifactId = "StandardLibrary"
         from(components["java"])
     }
-    publications.create<MavenPublication>("maven") {
-        groupId = "software.amazon.cryptography"
-        artifactId = "StandardLibrary"
-        from(components["java"])
-    }
-    repositories { mavenLocal() }
```

Comment:
I don't understand. Why are we pulling this out?
We remove `mavenLocal` almost everywhere else but not here?

Reply:
typo